cypress/integration/login.js

```diff
@@ -21,7 +21,9 @@ describe('Sourcebooks login', function() {
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Saulius Lipkevicius');
         cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.calendar--today').contains('24');

```

Comment:
Great job overall, but hard-coding value "24" into your test means it will no longer work tomorrow. Try to use a built-in JavaScript function to retrieve today's date whenever :)